src/actions/sources/select.js

```diff
@@ -142,17 +144,10 @@ export function selectLocation(location: Location) {
     dispatch(setSymbols(loadedSource.id));
     dispatch(setOutOfScopeLocations());
 
-    // If this is a new location and there is an active file search open,
-    // then update the results
-    const isLocationChanged =
-      !previousLocation || previousLocation.sourceId !== location.sourceId;
-    if (isLocationChanged) {
-      const isFileSearchOpen = getActiveSearch(getState()) === "file";
-      const fileSearchQuery = getFileSearchQuery(getState());
-      if (isFileSearchOpen && fileSearchQuery) {
-        const editor = getEditor();
-        dispatch(searchContents(fileSearchQuery, editor));
-      }
+    // If a new source is selected update the file search results
+    const newSource = getSelectedSource(getState());
+    if (!currentSource || (newSource && currentSource.id != newSource.id)) {
```

Comment:
I tried doing `(currentSource != newSource)` to check if the source was changed but that was evaluating to true even when the source wasn't changed.  So I used this instead